doc/manual/manual.tex

```diff
@@ -1315,14 +1316,233 @@ \subsection{Numerical methods}
+  +\alpha \bar\rho T' (\mathbf u \cdot \mathbf g)
+  \quad
+  \textrm{in $\Omega$}.
+\end{multline}
+
+
+
+\subsubsection{The truncated anelastic liquid approximation (TALA)}
+
+The \textit{Truncated Anelastic Liquid Approximation (TALA)} further simplifies
+the ALA by assuming that the variation of the density due to pressure variations
+is small, i.e., that
+\begin{align*}
+  \rho(p,T) \approx 
+  \bar\rho 
+  + \frac{\partial \rho(\bar p,\bar T)}{\partial T} T'.
+\end{align*}
+This does not mean that the density is not pressure dependent -- it will, for
+example, continue to be depth dependent because the hydrostatic pressure grows
+with depth. It simply means that the deviations from the reference pressure are
+assumed to be so small that they do not matter in describing the density.
+Because the pressure variation is induced by the flow field (the static
+component pressure is already taken care of by the hydrostatic pressure), this
+assumption in essence means that we assume the flow to be very slow, even beyond
+the earlier assumption that we can neglect inertial terms when
+deriving~\eqref{eq:stokes-1}--\eqref{eq:stokes-2}.
+
+This further assumption then
+transforms~\eqref{eq:stokes-ALA-1}--\eqref{eq:stokes-ALA-2} into the following
+equations:
+\begin{align}
+  \label{eq:stokes-TALA-1}
+  -\nabla \cdot \left[2\eta \left(\varepsilon(\mathbf u)
+                                  - \frac{1}{3}(\nabla \cdot \mathbf u)\mathbf 1\right)
+                \right] + \nabla p' &=
+  \frac{\partial \rho(\bar p,\bar T)}{\partial T} T' \mathbf g
+  & \qquad
+  & \textrm{in $\Omega$},
+  \\
+  \label{eq:stokes-TALA-2}
+  \nabla \cdot (\bar\rho \mathbf u) &= 0
+  & \qquad
+  & \textrm{in $\Omega$}.
+\end{align}
+
+
+\subsubsection{The extended Boussinesq approximation (EBA) and Boussinesq
+approximation (BA)}
+
+If we further assume that the reference density is constant, $\bar\rho(\bar
+p,\bar T)=\rho_0$, -- in other words, the density does not only not depend on
+the pressure variations $p'$ as assumed in the TALA, but also does not depend on
+the much larger hydrostatic pressure $\bar p$ nor on the reference temperature
+$\bar T$ -- then we can further simplify the mass conservation equations of the
+TALA to $\nabla \cdot \mathbf u=0$. We then obtain the following set of
+equations that also uses the incompressibility in the definition of the viscous
+strain:
+\begin{align}
+  \label{eq:stokes-EBA-1}
+  -\nabla \cdot \left[2\eta \varepsilon(\mathbf u)
+                \right] + \nabla p' &=
+  \frac{\partial \rho(\bar p,\bar T)}{\partial T} T' \mathbf g
+  & \qquad
+  & \textrm{in $\Omega$},
+  \\
+  \label{eq:stokes-EBA-2}
+  \nabla \cdot \mathbf u &= 0
+  & \qquad
+  & \textrm{in $\Omega$}.
+\end{align}
+
+These equations are identical in form for the \textit{Boussinesq approximation
+(BA)} which only differs from the EBA in neglecting compressibility effects in
+the temperature equation (shear heating and simplified adiabatic heating).
```

Comment:
I think it would be useful to state the temperature equation here for both the EBA and the BA.